gst/gstaravis.c

```diff
@@ -66,9 +66,19 @@ enum
   PROP_PACKET_RESEND,
   PROP_FEATURES,
   PROP_NUM_ARV_BUFFERS,
-  PROP_USB_MODE
+  PROP_USB_MODE,
+  PROP_SET_TRIGGER,
```

Comment:
Please rename set_trigger to just trigger, and make the property take a string instead of a boolean.